### ISSUE TITLE
Cannot give readonly access to a group

### ISSUE THREAD

**Issue by arifkara86** [grokability/snipe-it#9054]
Hello,

I'm trying to create a group that can only view.
Edit Group ->
- Assets
--- Appearance
--- Audit
--- View Desired Assets
- Accessories
--- Appearance
- Consumables
--- Appearance
.... etc.
When I grant view-only permissions, users in this group see either a single associated inventory or none at all. Likewise, users in the admin group do not have a list of all inventory on the List All page.
In other words, only super admin users can manage all inventory.

I would like to be able to create user groups that can see all these changes, but that is not possible.

**Comment by snipe** [comment]
Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?

:) Thanks!

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)
